doc/in-progress/sui-wallet-with-ledger.md

```diff
@@ -0,0 +1,117 @@
+
+##  View your account balance and assets
+
+You can view the balance for an account in Sui Wallet on the **Coins** tab. The coins and SUI stake displayed are for only the selected account address. To view the balance for a different account address, select the account from the drop-down list near the top of the wallet.
+
+To view the coins and tokens in your account that you imported from your Ledger device, select an account that displays *LEDGER* next to it. To view the NFTs for the account, click the **NFTs** tab in Sui Wallet.
+
+## Receive digital assets using Sui Wallet
+
+To receive cryptocurrency or other digital assets such as NFTs in Sui Wallet, select the account to receive the assets on the **Coins** tab, then click the copy icon to copy the address.
+
+Before you use an account from your Ledger device to receive an asset, confirm that the account is connected to your device.
+
+**To receive digital assets to your Ledger account using Sui Wallet** 
+1. Open Sui Wallet and click the Settings menu (the three bars displayed in the top-right corner).
+1. Click **Accounts**.
+1. Click the down arrow next to the Ledger account to receive the asset, then click **Verify Ledger connection**. Make sure that the display shows **Ledger is connected** before you initiate the transaction to receive an asset.
+1. Send the digital asset to the address for the Ledger account.
+
+## Send digital assets using Sui Wallet
+
+You can use Sui Wallet to send a digital asset from your Ledger account.
+
+1. Open Sui Wallet and click the Settings menu (the three bars displayed in the top-right corner).
+1. Click **Accounts**.
+1. Click the down arrow next to the Ledger account to receive the asset, then click **Verify Ledger connection**. Make sure that the display shows **Ledger is connected** before you initiate the transaction to send an asset.
+1. Click the **Coins** tab in Sui Wallet, then click **Send**.
+1. Enter the amount of SUI to send and the address of the recipient, then click **Review**.
+1. Review the details of the transaction to confirm that they are accurate. If they are, click **Send now**. If not, click **Back** to make changes, or click **X** to cancel.
+1. Your Ledger device displays **Transfer SUI**. Press the Right button on the device to display the following details. Press it twice to display the full addresses used. You should verify that the addresses are correct before you finalize the transaction. Confirm that the address on the device matches the address displayed in Sui Wallet.
+   * The address the transaction is from.
+   * The recipient address.
+   * The amount to send.
+   * Sign transaction.
+   * Confirm
+1. Press both buttons while it displays Confirm to approve the transaction.
```

Comment:
We might want to add another step in case the user wants to reject the transaction, in which case they need to go to the right one more time and hit "Reject"

Reply:
added, ty